ivadomed/utils.py

```diff
@@ -661,6 +662,21 @@ def segment_volume(fname_model, fname_model_metadata, fname_image, fname_roi=Non
     # Define device
     device = torch.device("cpu")
 
+    # Check if model folder exists
+    if os.path.isdir(folder_model):
+        # Check if model and model metadata exist
+        fname_model = os.path.join(folder_model, 'model.pt')
```

Comment:
is it an official convention in IVADOMED to have all models called "model.pt"? If it is called model_bob.pt, this line will fail. 

- If it is an official convention, i would remove the explicit file name here and move it to a separate class, e.g. `ivadomed.metadata.get_model()`, as done [in SCT](https://github.com/neuropoly/spinalcordtoolbox/blob/master/spinalcordtoolbox/metadata.py#L191). All the appropriate i/o checks could be done in this dedicated module. We can address this refactoring in a separate PR though (and leave the current code as is for this PR).
- If it not an official convention, I would maybe suggest to name the model with the same basename as the folder: `model_bob/model_bob.pt`. This might prove useful in the future, if we manually copy models during debugging/testing, to avoid confusion (e.g. if all models are called "model.pt, robert might think he plays with the model from bob, but in fact it is the model from ariane. calling it model_bob.pt clarifies it). Again, inspired by the BIDS convention.

Reply:
It may be a good time to discuss the official convention in ivadomed :-)

I do like the Bob solution! So should we have `model_bob/model_bob.json` and `model_bob/model_bob.pt` for the metadata and the model, resp?